src/scenes/start-scene.js

```diff
@@ -142,12 +143,40 @@ export class StartScene extends Phaser.Scene {
             }
         });
 
-
+        this.randomizeButton = new TextButton(this, 50, 300, 'RANDOMIZE SEATING', {
+            onClick: () => {
+                // Make an array all the seats and randomly pick them off for each player
+                this.removeInactivePlayers() // FIXME normally for every player, there are ~2 additional inactive players. This manages to remove all of them
+                let emptySeats = []
+                this.seats.forEach(seat => {
+                    emptySeats.push(seat);
+                });
+                console.log('players: ', this.playersLabel.playerLabels) // FIXME used for debugging inactive player problem
+                this.playersLabel.playerLabels.forEach(label => {
+                    let randomIndex = Math.floor(Math.random()*emptySeats.length);
+                    let next_seat = emptySeats.splice(randomIndex, 1)[0]
+                    next_seat.add(label);
+                });
+            }
+        });
+        this.add.existing(this.randomizeButton);
+        this.randomizeButton.setVisible(false);
```

Comment:
Could be worth keeping "randomized" and allow clicking again if you want another random order.

Reply:
Oh yeah I didn't even notice that line because the button actually never disappeared... which I don't really understand. But I removed that line anyway

Reply:
Oh that's right, the visibility is set to false until the host joins and then it's set to true. So the button always stays